staging/src/k8s.io/kubectl/pkg/cmd/top/top_node.go

```diff
@@ -65,7 +65,10 @@ var (
 		  kubectl top node
 
 		  # Show metrics for a given node
-		  kubectl top node NODE_NAME`))
+		  kubectl top node NODE_NAME
+		  
+		  # Show metrics based on the capacity for a given node
+		  kubectl top node NODE_NAME --show-capacity`))
```

Comment:
Usually we tend to add examples for non-obvious flags, this particular cases doesn't add anything more than what this particular flag description.

Reply:
Thank you for your review. It makes sense for me as well. 
Just I reckoned this example is reasonable compared to `kubectl top pod` examples. Such like `kubectl top pod POD_NAME --containers`. The `--show-capacity` flag makes the output switch to another mode like above `--containers`, so I thought it helps adding this example to `kubectl top node` for more kind example usage. So I'd like to add this example for better user experience.

Reply:
I'm not convinced that we should add examples just to have examples, which is what you're saying.

Reply:
@soltysh Thank you for your review. I got understand you. I also don't want to push this hard. I closed this ticket.